### DISCUSSION TITLE
Authenticate with different accounts based on directory

### DISCUSSION THREAD

**Discussion by quulah** [zbirenbaum/copilot.lua#253] [Q&A]
Would there be a way to authenticate with a specific account based on directory?

**Comment by MunifTanjim** [comment]
The auth information is usually stored in `~/.config/github-copilot/hosts.json` file.

Ref:
- https://github.com/zbirenbaum/copilot.lua/discussions/120

You can potentially do this:
- Log in as account `A`, rename that file to `hosts-A.json`
- Log in as account `B`, rename that file to `hosts-B.json`

Now if you want to use account `A`, copy `hosts-A.json` to `hosts.json`. You get the idea.

But I'm not sure if it'll work if you're using multiple neovim instance at the same time. Maybe try it and see how it goes.

The `copilot` language server doesn't support this officially. So this is just a workaround.
